spec/api/v2/token_spec.rb

```diff
@@ -3,21 +3,55 @@
 describe '/v2/token' do
 
   describe 'get token' do
+    before :all do
```

Comment:
More than that - you already have factory and you can use

```ruby
let(:user) { create(:user) }
```

just after describe